src/otx/core/config/export.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Config data type objects for export method."""
+
+from dataclasses import dataclass
+from typing import Tuple
+
+from otx.core.types.export import OTXExportPrecisionType, OTXExportFormatType
+
+
+@dataclass
+class ExportConfig:
```

Comment:
In my thought, it's better to set this class at same level as `TrainConfig` and pass it to `export(cfg)` in "src/otx/core/engine/export.py" (may need to add other attribute such as `model` for that). How do you think about?